pl/tutorials/tour/singleton-objects.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: tutorial
+title: Obiekty singleton
+
+disqus: true
+
+tutorial: scala-tour
+num: 12
+language: pl
+
+tutorial-next: xml-processing
+tutorial-previous: pattern-matching
+---
+
+Metody i wartości które nie są powiązane z konkretną instancją [klasy](classes.html) należą do *obiektów singleton*, określanych za pomocą słowa kluczowego `object` zamiast `class`.
+
+```
+package test
+
+object Blah {
+  def sum(l: List[Int]): Int = l.sum
+}
+```
+
+Metoda `sum` jest dostępna globalnie i można się do niej odwołać lub importować jako `test.Blah.sum`.
+
+Obiekty singleton są swego rodzaju skrótem do definiowania pojedynczej instancji klasy, która nie powinna być bezpośrednio tworzona i która sama w sobie stanowi referencję do tego obiektu, jakby była określona jako `val`. 
+
+Obiekt singleton może rozszerzać klasę lub cechę. Przykładowo [klasa case](case-class.html) bez [parametrów typu](generic-class.html), domyślnie generuje obiekt singleton o tej samej nazwie, który implementuje cechę [`Function*`](http://www.scala-lang.org/api/current/scala/Function1.html).
+
+## Companion ##
+
+Duża część obiektów singleton nie istnieje samodzielnie, ale jest powiązana z klasą o tej samej nazwie. Obiekt singleton generowany dla klasy case jest tego przykładem. Kiedy tak się dzieje, obiekt singleton jest zwany *obiektem companion* (co oznacza, że jest _towarzyszem_ tej klasy).
```

Comment:
I though about `obiekt towarzysz` which sounded a bit awkward but I like `obiekt towarzyszący`, I think we can use this translation.